vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs

```diff
@@ -192,7 +194,7 @@ type internal FSharpCompletionProvider
                 declarationItemsCache.Add(completionItem.DisplayText, declItem)
                 results.Add(completionItem))
 
-            if results.Count > 0 && not declarations.IsForType && not declarations.IsError then
+            if results.Count > 0 && not declarations.IsForType && not declarations.IsError && List.isEmpty qualifyingNames then
```

Comment:
I think `isNil` is used in the compiler and FCS exclusively. We should use normal functions in FSharp.Editor.